src/test/java/org/terasology/genome/DiscreteBreedingAlgorithmTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.genome;
+
+import org.junit.jupiter.api.Test;
+import org.terasology.genome.breed.BreedingAlgorithm;
+import org.terasology.genome.breed.DiscreteBreedingAlgorithm;
+import org.terasology.genome.breed.mutator.GeneMutator;
+import org.terasology.genome.breed.mutator.VocabularyGeneMutator;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid `*` imports. There is a setting in IntelliJ Idea which auto converts multiple imports to `*` imports, you can bump up that number to some high value (like 999) to avoid these auto converts.